src/meta/src/manager/catalog/mod.rs

```diff
@@ -317,7 +317,19 @@ where
                 .collect_vec();
             let users_need_update = Self::update_user_privileges(&mut users, &objects);
```

Comment:
Do we need to call `update_user_privileges` for connections?

Reply:
We don't have any privileges related to connection yet.